libraries/chain/include/eosio/chain/finality/finalizer_policy.hpp

```diff
@@ -2,13 +2,37 @@
 
 #include <eosio/chain/types.hpp>
 #include <eosio/chain/finality/finalizer_authority.hpp>
+#include <fc/container/ordered_diff.hpp>
 
 namespace eosio::chain {
 
+   using finalizers_differ = fc::ordered_diff<finalizer_authority>;
+   using finalizers_diff_t = fc::ordered_diff<finalizer_authority>::diff_result;
+
+   struct finalizer_policy_diff {
```

Comment:
Not sure if `finalizer_policy_diff` is needed, as `generation` and `threshold` are not really differed. Only finalizers are differed.

Reply:
Provides a struct for easy `fc::pack` into `instant_finality_extension`.